### ISSUE TITLE
VSync not working since v1.1.200

### ISSUE THREAD

**Issue by olvlvl** [Ryujinx/Ryujinx#3534]
### What's the issue you encountered?

Hi everyone, I've recently updated from v1.1.199 and the VSync option is no longer working for me. Whether it's on or off, it's screen tearing galore. I tried a clean install, same issue. I reverted to v1.1.199 and vsync works as expected.

Whether I'm using native resolution or x4 the issue persists, although it's less pronounced with lower resolution. Also, whether I'm using OpenGL or Vulkan the issue persists.

CPU utilization is about 20%, and GPU utilization is about 40%.

It's too bad because performance is vastly improved with the new version :(

### How can the issue be reproduced?

For me, updating to v1.1.201 caused the issue. I've updated to v1.1.203 but the issue remains.

### Log file

```
00:00:00.003 |I| Configuration LogValueChange: EnableFileLog set to: True
00:00:00.037 |I| Configuration LogValueChange: ResScale set to: 4
00:00:00.041 |I| Configuration LogValueChange: ResScaleCustom set to: 1
00:00:00.041 |I| Configuration LogValueChange: MaxAnisotropy set to: -1
00:00:00.041 |I| Configuration LogValueChange: AspectRatio set to: Fixed16x9
00:00:00.042 |I| Configuration LogValueChange: BackendThreading set to: Auto
00:00:00.043 |I| Configuration LogValueChange: GraphicsBackend set to: Vulkan
00:00:00.043 |I| Configuration LogValueChange: PreferredGpu set to: 0x10DE_0x2208
00:00:00.044 |I| Configuration LogValueChange: EnableDockedMode set to: True
00:00:00.044 |I| Configuration LogValueChange: EnableVsync set to: True
00:00:00.045 |I| Configuration LogValueChange: EnableShaderCache set to: True
00:00:00.045 |I| Configuration LogValueChange: EnableTextureRecompression set to: False
00:00:00.045 |I| Configuration LogValueChange: EnablePtc set to: True
00:00:00.045 |I| Configuration LogValueChange: EnableInternetAccess set to: False
00:00:00.045 |I| Configuration LogValueChange: EnableFsIntegrityChecks set to: True
00:00:00.045 |I| Configuration LogValueChange: FsGlobalAccessLogMode set to: 0
00:00:00.045 |I| Configuration LogValueChange: AudioBackend set to: SDL2
00:00:00.045 |I| Configuration LogValueChange: AudioVolume set to: 1
00:00:00.046 |I| Configuration LogValueChange: MemoryManagerMode set to: HostMappedUnsafe
00:00:00.046 |I| Configuration LogValueChange: ExpandRam set to: False
00:00:00.046 |I| Configuration LogValueChange: IgnoreMissingServices set to: False
00:00:00.048 |N| Application PrintSystemInfo: Ryujinx Version: 1.1.203
00:00:00.052 |N| Application Print: Operating System: Microsoft Windows 10.0.22000 (X64)
00:00:00.052 |N| Application Print: CPU: 12th Gen Intel(R) Core(TM) i5-12600 ; 12 logical
00:00:00.052 |N| Application Print: RAM: Total 16175 MB ; Available 11236 MB
00:00:00.054 |N| Application PrintSystemInfo: Logs Enabled: Info, Warning, Error, Guest, Stub
00:00:00.054 |N| Application PrintSystemInfo: Launch Mode: Portable
```

### Environment?

- Ryujinx version: 1.1.199</br>
- Game version: Xenoblade 1, 2, 3</br>
- System Specs:
  - OS: Windows 11
  - CPU: i5-12600
  - GPU: NVIDIA RTX 3080
  - RAM: 16GB
- Applied Mods : No

### Additional context?

I set my desktop refresh rate to 30 hz so I can enable TruMotion Smooth on my LG C9. But the issue persists no matter the refresh rate, I tried 60 and 120 too.

**Comment by MutantAura** [comment]
Ryujinx vsync is "guest" vsync, or vsync of the emulated switch. It is not what you may be more familiar with "host" vsync which is about screen tearing and locking your game to the monitors refresh rate.
You need to enable vsync or VRR in your driver if you want to remove tearing. Ryu vsync in laymans terms is just a frame limiter.

**Comment by gdkchan** [comment]
Let's keep this open, it's something that can (and should IMO) be fixed on our end changing the present mode on Vulkan based on the VSync setting.

**Comment by olvlvl** [comment]
I've added the application to the NVidia Control Panel and set VSync there. It's fine now but I never had to do that for previous versions.

**Comment by gdkchan** [comment]
On Vulkan, it will have tearing regardless of the emulator setting if VSync is disabled on the driver right now. On OpenGL, the behavior should not have changed, not sure why it changed on OpenGL for you.

**Comment by coolllman** [comment]
But mario kart and others crash in menu, when click gran pri, when vsync ryujinx off and vsync on in nvidia control panel

**Comment by coolllman** [comment]
When vsync ryujinx on nvidia vsync not work

**Comment by Immersion95** [comment]
I also had problem with AMD VSync + OpenGL but it's fixed on latest driver 22.8.1.

Unfortunately it never worked with Vulkan, is there plans to solve it ? What about Host VSync implemented in the program ? Thanks !

**Comment by Kein** [comment]
Is there any way to configure the internal value at which caps is applied when "fake" vsync used in Ryu? I'm fine with it being a limiter, I just want different value than 60.

**Comment by dortamiguel** [comment]
The vsync won't work either on mac

**Comment by TSRBerry** [comment]
The macOS release had an issue which was later fixed here: #4379

**Comment by dortamiguel** [comment]
I'm using the latest download from the website, but when using an external monitor with 60hz the tearing is very bad

**Comment by TSRBerry** [comment]
I know, that's because there is no new release available yet. Sorry!

**Comment by dortamiguel** [comment]
I tried building it from latest on my local machine with `dotnet build -c Release -o build` but if I try to start the program like this `./build/Ryujinx` it gives me an error. Is it posible to produce that .app for mac with a build? I can't find this on the wiki

**Comment by siegmund-heiss-ich** [comment]
This script downloads and builds the latest source code, builds the app and saves it as `Ryujinx_custom.app` in your Application folder. The official release won't be deleted.

```
cd /tmp
git clone https://github.com/Ryujinx/Ryujinx
cd Ryujinx
./distribution/macos/create_macos_release.sh ./ ./distribution/macos/temp ./distribution/macos/output ./distribution/macos/entitlements.xml 1.1.1 0
cp -r ./distribution/macos/output/Ryujinx.app /Applications/Ryujinx_custom.app
rm -rf /tmp/Ryujinx 
```

**Comment by dortamiguel** [comment]
@siegmund-heiss-ich thank you! it worked really well, now the tearing is fixed with latest :)
